### DISCUSSION TITLE
The users morphedByMany relation gives error

### DISCUSSION THREAD

**Discussion by sruthitechfriar** [spatie/laravel-permission#2453] [Q&A]
The morphedByMany relation is not working as expected

$roles = Role::with('users')->get();
dd($roles);

"Class name must be a valid object or a string"

**Comment by cesarreyes3** [comment]
It's an error in your implementation, read other issues/discussions about it

**Comment by drbyte** [comment]
Perhaps you've got fields in your database or properties on your models which conflict with this package?
https://spatie.be/docs/laravel-permission/v5/prerequisites
